app/dts/bindings/behaviors/zmk,behavior-smart-interrupt.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Smart Interrupt Behavior
+
+compatible: "zmk,behavior-smart-interrupt"
+
+include: zero_param.yaml
+
+properties:
+  bindings:
+    type: phandle-array
+    required: true
+  shared-key-positions:
```

Comment:
Not sure I love the term `shared` for this? Maybe `ignored-key-positions` or `continuation-key-positions`?